.github/workflows/brew-publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Homebrew
+on:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Update Homebrew formulae
+        uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
+          org: pyroscope
+          token: ${{ secrets.TOKEN }}
```

Comment:
Can you call it something else, like `secrets.GITHUB_HOMEBREW_TOKEN`? I just want to make sure that when we have a bunch of secrets in the repo it's still clear what each token does.

Reply:
```suggestion
          token: ${{ secrets. GITHUB_HOMEBREW_TOKEN }}
```

Reply:
LGTM